jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ExceptionUtil.java

```diff
@@ -107,9 +110,29 @@ public static void ifExceptionThrowUnchecked(Throwable throwable)
             throw runtimeException;
         if (throwable instanceof Error error)
             throw error;
+        if (throwable instanceof IOException ioException)
+            throw new UncheckedIOException(ioException);
         throw new RuntimeException(throwable);
     }
 
+    /**
+     * Convert or cast a {@link Throwable} to an unchecked {@link RuntimeException}.
+     * @param cause The {@link Throwable} to that cause.
+     * @throws Error If the passed {@link Throwable} is an {@link Error}.
+     * @return RuntimeException Otherwise, if the passed {@link Throwable} is not null.
+     */
+    public static RuntimeException asRuntime(Throwable cause) throws Error
```

Comment:
I'd prefer `asRuntimeException()`, but then again how many times it this used? Only two.
I'd rather remove it, and handle the two cases, since this method has this strange behavior that most of the time does a conversion, but throws `Error`.